newrelic/example_test.go

```diff
@@ -43,3 +36,48 @@ func ExampleNewExporter() {
 	}
 	global.SetTraceProvider(tp)
 }
+
+func ExampleNewExportPipeline() {
+	// Include environment in resource.
+	r := resource.New(
+		kv.String("environment", "production"),
+		standard.ServiceNameKey.String("My Service"),
+	)
+
+	// Assumes NEW_RELIC_API_KEY contains your New Relic API key. This will
```

Comment:
Possibly worth specifying that `NEW_RELIC_API_KEY` should an environment variable (I assume).

Reply:
Updated.